.github/workflows/productionDeploy.yaml

```
@@ -0,0 +1,95 @@
+  SESSION_SECRET: ${{ secrets.SESSION_SECRET }}
+  NODE_ENV: production
+
+jobs:
+  check-test-status:
+    runs-on: ubuntu-latest
+    outputs:
+      test_status: ${{ steps.check_status.outputs.status }}
+    steps:
+      - name: Check Test Stage Status
+        id: check_status
+        run: |
```

actionlint (1.7.4)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)